src/App/App.js

```diff
@@ -28,6 +29,36 @@ class App extends Component {
 			})
 	}
 
+	componentDidUpdate(prevProps, prevState) {
+		if (this.state.isLoggedIn !== prevState.isLoggedIn) {
+			getUserRatings(this.state.id)
+				.then(userRatings => {
+					this.setState({ userRatings: userRatings.ratings })
+				})
+				.catch(error => console.log(error))
+		}
+	}
+
+	submitRating = (userId, userRating, movieId, event) => {
+		if (userRating) {
+			deleteRating(userId, this.findRatingId(movieId))
+		}
+		userRating = +userRating
+		event.preventDefault()
+		postRating(userId, userRating, movieId)
+			.then(rating => {
+				this.setState({ userRatings: [...this.state.userRatings, rating.rating] })
+			})
+			.catch(error => console.log(error))
+	}
+
```

Comment:
This is great, and you already know this but we'll need to provide some more user-friendly error handling.